addons/portal/__init__.py

```diff
@@ -1,6 +1,21 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+# Updating mako environement in order to be able to use slug
+try:
```

Comment:
just to be sure, the try/except is there in case of `mako_template_env` does not exist ?

Reply:
Yep exactly